### ISSUE TITLE
connect edit_parameters test to different signal

### ISSUE THREAD

**Issue by gassmoeller** [geodynamics/aspect#5078]
Following #5070 the test `edit_parameters` no longer does what it did before, because it was connected to a signal that is no longer triggered. We need to fix this by connecting the test to a different signal that triggers every timestep.

**Comment by tjhei** [comment]
Yes, I agree. Do you think it makes sense to leave it as a CFL change instead of something more interesting like the change in boundary conditions (what it did before)?

I was thinking we could introduce a "begin time step" signal and use this here.

**Comment by gassmoeller** [comment]
Yes, I think "begin time step" would be a very useful signal to have. 

> Do you think it makes sense to leave it as a CFL change instead of something more interesting like the change in boundary conditions (what it did before)?

Interesting question, I think I would be fine with either.  Both may be useful to showcase.
